sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseEditAuthorization.java

```diff
@@ -0,0 +1,65 @@
+package de.symeda.sormas.backend.caze;
+
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import javax.ejb.EJB;
+import javax.ejb.LocalBean;
+import javax.ejb.Stateless;
+
+
+import de.symeda.sormas.api.user.UserRole;
+import de.symeda.sormas.backend.facility.Facility;
+import de.symeda.sormas.backend.infrastructure.PointOfEntry;
+import de.symeda.sormas.backend.region.Community;
+import de.symeda.sormas.backend.region.District;
+import de.symeda.sormas.backend.region.Region;
+import de.symeda.sormas.backend.user.User;
+import de.symeda.sormas.backend.user.UserService;
+
+@Stateless (name = "CaseEditAuthorization")
+@LocalBean
+public class CaseEditAuthorization {
+	
+	@EJB
+	private UserService userService;
+	@EJB
+	private CaseService caseService;
+
+	public Boolean caseEditAllowedCheck(String caseUuid) {
```

Comment:
@sergiupacurariu You need to check for null for most of the case properties. You can use DataHelper.equal then you don't have to care about it

Reply:
changed